pages/collections/[slug].vue

```diff
@@ -68,13 +101,24 @@ const parsedRelatedCollections = computed(() => {
     return {
       ...item,
       to: `/${item.uri}`, // remove 'collection/' from uri
-      sectionHandle: 'ftvaEventSeries',
       category: 'collection',
+      bylineOne: item.richText,
```

Comment:
should we use text prop here?

Reply:
The byline had the grey text so I used that, but I could switch it and make the grey color a page level style.

Reply:
Lets keep this prop for now. I will check in with UX later. Even though we use v-html for byline in card  meta I would prefer to use text prop which uses richtext . But since it is used in BlockCardWithImage  context I can go either ways.